src/ILCompiler.Compiler/src/Compiler/CompilationModuleGroup.cs

```diff
@@ -27,8 +27,12 @@ protected CompilationModuleGroup(CompilerTypeSystemContext typeSystemContext, IC
             _rootProvider = rootProvider;
         }
 
-        public abstract bool IsTypeInCompilationGroup(TypeDesc type);
-        public abstract bool IsMethodInCompilationGroup(MethodDesc method);
+        public abstract bool IsInCompilationGroup(TypeDesc type);
```

Comment:
Usually there is no need to repeat the type name in a method. I would recommend simply having:
```HasType(TypeDesc type)``` and ```HasMethod(MethodDesc method)``` and do not use overloading for that.